src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/THM_wellbore_displacement.py

```diff
@@ -0,0 +1,73 @@
+import matplotlib
```

Comment:
All python scripts did not work properly and cannot generate expected figures and please double check them.

Reply:
see the comment above